### ISSUE TITLE
TSCONFIG: Error Unknown Option: typeRoots

### ISSUE THREAD

**Issue by martov1** [alm-tools/alm#221]
Hi there!,

I'm trying to use alm on an angular 2 project built with angular-cli and it seems that alm is having some trouble with the property "typeRoots" in tsconfig.json

this is the tsconfig.json
```

{
  "compilerOptions": {
    "declaration": false,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "lib": ["es6", "dom"],
    "mapRoot": "./",
    "module": "es6",
    "moduleResolution": "node",
    "outDir": "../dist/out-tsc",
    "sourceMap": true,
    "target": "es5",
    "typeRoots": [
      "../node_modules/@types"
    ]
  }
}

```

It works fine when using the angularCLI.

I also noticed that in this case if i'm using angularCli to process my typescript I have no way of stopping ALM to try to transform my ts files into JS, right?

**Comment by basarat** [comment]
> I have no way of stopping ALM to try to transform my ts files into JS, right?

Wrong. You can use `compileOnSave` set to false : https://basarat.gitbooks.io/alm/content/config/tsconfig.html#compileonsave

**Comment by martov1** [comment]
Thanks c:

**Comment by basarat** [comment]
Sorry about that. Thanks for pointing out the error. Fixed with `2.2.2` :rose:

**Comment by martov1** [comment]
Thank you for fixing it!.
